script/Ciel/Day 13/13_01_01_CIEL13_3_2.txt

```diff
@@ -106,37 +112,39 @@ The Black Key is pinning my shadow down, and it seems like that's the reason why
 [sha:91cfc68b911c35b67f3cc4268e1a31e86c5caa94]{
 -- Page 352, Offset 34250.
 -- 両足に力を込める。
-I pour my strength into my legs.
+I strain my legs as hard as I can.
 }
 [sha:0a519b9b38e0b125980dc7b47ab6fe259805ae56]{
 -- Page 352, Offset 34251.
 -- しかし、１ミリも動かない。
-However, they don't budge even a single milimeter.
+Try as I might, they don't budge so much as a millimeter.
 }
 [sha:51575846a60b816efe0530c5d1752e1caadc270a]{
 -- Page 352, Offset 34252.
 -- トラバサミにかかった犬の姿が脳裏に浮かぶ。
-The image of a dog caught in a steel trap comes to mind.
+// Slight tampering with the 脳裏に浮かぶ but ehh it's the idea
+I feel like a dog caught in a bear trap.
 }
 [offset:34253]{
 -- Page 352, Offset 34253.
 -- そこへ、
-Over there in the distance―――
+And then―――
 }
 [sha:9adecb9fe985bd272b3468d6adec7745f222c7f9]{
 -- Page 353, Offset 34254.
 -- 「浅はかですね。逃げるにしても、もう少し上手いやり方があったでしょうに」
-"How foolish. If you're going to run, then you should at least come up with a better escape plan."
+"A disappointing display.
+ If you were going to run, then surely there was a better way you could have done it."
 }
 [sha:aad06eea8c3cdfc19d019d481664194d2c3b63d6]{
 -- Page 353, Offset 34255.
 -- “弓”の異名をとる代行者が現れた。
-The Executor known as the 'Bow' appears before me.
+The Executor nicknamed 'Bow' appears before me.
 }
 [sha:68ff6429413aec83978735970cfb92886d138660]{
 -- Page 354, Offset 34256.
 -- 「先―――輩」
-"Sen―――pai."
+"Sen――pai."
 }
 [sha:5058e534624024a4f527b61500a6848b9aeea710]{
 -- Page 354, Offset 34257.
```

Comment:
While "rain down judgment" is a bible phrase and all, it is more commonly just "bring judgment" there anyway. I feel like this one feels a bit forced.